portalnet/src/overlay/protocol.rs

```diff
@@ -518,48 +518,42 @@ where
 
     /// Offer is sent in order to store content to k nodes with radii that contain content-id
     /// Offer is also sent to nodes after FindContent (POKE)
-    pub async fn send_offer(
+    pub async fn send_wire_offer(
         &self,
-        content_keys: Vec<RawContentKey>,
         enr: Enr,
+        content_keys: Vec<RawContentKey>,
```

Comment:
I would make this type `Vec<TContentKey>` because:
1. all calls to this function convert `Vec<TContentKey>` to `Vec<RawContentKey>` right before the call
2. we immediately convert it back to `Vec<TContentKey>`

Seems like we are doing some useless work